dask/array/linalg.py

```diff
@@ -641,6 +642,10 @@ def compression_matrix(data, q, n_power_iter=0, seed=None):
     n_power_iter: int
         number of power iterations, useful when the singular values of
         the input matrix decay very slowly.
+    recompute : bool
+        Should we compute data at each stage during computation?
```

Comment:
Stage -> power iteration? I'm not sure what the common term is here, but this links it strongly with the previous parameter.

Reply:
Yes, each power iteration, and also the first matmul/tsqr call (which is possibly more important).  I'm not sure that we can call this each power iteration.

Reply:
How about each use? Non-technical term, and leaves slack in definition if the implementation changes.